docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I really don't think you need this file. Why is it include? `.bat` files are typically only on Windows.

Reply:
I'm not sure. Since this is open-source, can anyone propose changes to the docs in the form of newly generated `.rst` files? If a person wanted to do this using a Windows machine, they would need to use this file? That's two questions. I need research it before removing it.

Reply:
Yes, anyone can propose changes to the docs, including `.rst` files. However, coremltools doesn't have native Windows support. So I don't think we need to worry about people trying to natively build our docs on Window.

Reply:
Good point! Removing the file. Thanks!